app/views/works/index.html.erb

```diff
@@ -0,0 +1,71 @@
+<h2> List of Works </h2>
+
+
+
+<table class="table">
+  <h4> Books </h4>
+  <thead>
+    <tr>
+      <th> Votes </th>
+      <th> Title </th>
+      <th> Creator </th>
+      <th> Published Year </th>
+      <th> Upvote </th>
+    </tr>
+  </thead>
+  <tbody>
+<tr> <% @books.each do |work| %>
+  <td> <%= work.votes.length %> </td>
+  <td><%= link_to "#{work.title}", work_path(work.id) %></td>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?